### PR TITLE
Bc 2219 registration process text changes

### PR DESCRIPTION
**Pull request by odalys-dataport**
# Description

https://ticketsystem.dbildungscloud.de/browse/BC-2219

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: controllers/registration.js [hunk 1]
```diff
@@ -433,6 +433,7 @@ router.get(['/registration/:classOrSchoolId', '/registration/:classOrSchoolId/:s
 			invalid,
 			secure,
 			correctID,
+			cloudConsentNecessary: res.locals.theme.cloud_consent_necessary
```
Comment (christian-darsow) [inline controllers/registration.js:436]: this should be now `res.locals.theme.consent_necessary`, right?

**Review by christian-darsow — CHANGES_REQUESTED** [review]

File: controllers/registration.js (hunk 1 above)
Comment (odalys-dataport) [reply controllers/registration.js:436]: Yup, you're right :D

**Review by christian-darsow — APPROVED** [review]

File: views/registration/registration.hbs [hunk 2]
```diff
@@ -37,20 +41,29 @@
...
+				<p>{{$t "registration._registration.text.nextStepRegistration" (dict "title" theme.theme_title)}}</p>
+			{{/if}}
 		</div>
 		<div id="infotext-16" class="hidden">
-			<p>{{$t "registration._registration.text.thankYouForTakingThisStepWithUs" (dict "shortTitle" theme.title)}}</p>
-			<p>{{$t "registration._registration.text.pleaseRegisterFirstAndAgreeToTerms" (dict "themetitle" theme.title)}}</p>
-			<p><strong>{{$t "global.text.attention" }}</strong> {{$t "registration._registration.text.registrationMustBeDoneByYou" }}</p>
+			{{#if cloudConsentNecessary}}
+				<p>{{$t "registration._registration.text.thankYouForTakingThisStepWithUs" (dict "shortTitle" theme.title)}}</p>
+				<p>{{$t "registration._registration.text.pleaseRegisterFirstAndAgreeToTerms" (dict "themetitle" theme.title)}}</p>
```
Comment (MartinSchuhmacher) [inline views/registration/registration.hbs:54]: `themetitle` seems to be wrong here, as in the key itself it requires `shortTitle`, did you check if that one works like it is implemented here?

**Review by MartinSchuhmacher — CHANGES_REQUESTED** [review]

File: views/registration/registration.hbs (hunk 2 above)
Comment (virgilchiriac) [reply views/registration/registration.hbs:54]: was the same before, only some white space is added

Comment (MartinSchuhmacher) [reply views/registration/registration.hbs:54]: I know, but if it was wrong before we should fix that as well ;)

**Review by MartinSchuhmacher — APPROVED** [review]

**Pull request merged**